orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarter.groovy

```diff
@@ -88,6 +91,14 @@ class DependentPipelineStarter implements ApplicationContextAware {
     def t1 = new Thread(new Runnable() {
       @Override
       public void run() {
+        if (user) {
+
+          def securityCtx = SecurityContextHolder.createEmptyContext()
+          // Using RunAsUserToken is an arbitrary Authentication objection - it's merely named the best for this use case.
```

Comment:
This also probably isn't the right "user", looks like it's coming out of a trigger and/or context variable.

https://github.com/spinnaker/orca/blob/master/orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/spring/DependentPipelineExecutionListener.groovy#L57

https://github.com/spinnaker/orca/blob/master/orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/tasks/StartPipelineTask.groovy#L76

Feels like doing something along the lines of what `AuthenticatedRequest.propagate()` is doing might be appropriate?

https://github.com/spinnaker/orca/blob/master/orca-core/src/main/groovy/com/netflix/spinnaker/orca/batch/adapters/TaskTasklet.groovy#L194

^^ is what we're doing in the TaskTasklet to propagate the auth context from a running execution.